src/app/pages/SearchFunction.ts

```diff
@@ -1,36 +1,39 @@
 import { Result } from '../../__types__';
 
-const MAX_RESULT = 20; // stop searching once we get more than 10 results
-const MAX_TEXT_LENGTH = 256; // get this many text as a preview for
+const MAX_RESULT = 20; // stop searching once we get 20 results
```

Comment:
I would drop this down to 10...when you get down to the 20th result, it's a bit of a stretch for what I'm actually searching for